libs/handlers/src/createLambdaHandler.ts

```diff
@@ -28,18 +33,24 @@ export interface LambdaHandlerConfig<TEvent = unknown, TResult = unknown> {
  *   },
  *   {
  *     middlewares: [customMiddleware()],
+ *     loggerOptions: { level: 'INFO', serviceName: 'my-service' },
  *   }
  * );
  * ```
  */
 export function createLambdaHandler<TEvent = unknown, TResult = unknown>(
   handler: Handler<TEvent, TResult>,
-  config?: LambdaHandlerConfig<TEvent, TResult>,
+  config: LambdaHandlerConfig<TEvent, TResult>,
 ): MiddyfiedHandler<TEvent, TResult, Error, Context> {
   const middyHandler = middy<TEvent, TResult, Error, Context>(handler);
 
-  // Apply custom middlewares
-  if (config?.middlewares && config.middlewares.length > 0) {
+  middyHandler.use(
```

Comment:
I'm wondering, because the logger instance is created outside of the handler (within the `createHandler`), it's effectively a singleton? 

Or are you thinking of other use-cases.